homeassistant/components/yeelight/light.py

```diff
@@ -579,6 +583,20 @@ def light_type(self):
         import yeelight
         return yeelight.enums.LightType.Ambient
 
+    @property
+    def min_mireds(self):
+        """Return minimum supported color temperature."""
+        return kelvin_to_mired(6500)
```

Comment:
Please apply (`self._min_mireds) this calculation at the constructor or assign the result in mireds here.